.github/workflows/bottle.yml

```diff
@@ -0,0 +1,258 @@
+name: bottle
+
+on: workflow_call
+
+jobs:
+  bottle:
+    runs-on: ${{ matrix.platform.os }}
+    defaults:
+      run:
+        working-directory: tea.xyz/var/pantry
+    strategy:
+      matrix:
+        platform:
+          - os: macos-11
+            name: darwin+x86-64
+          - os: ubuntu-latest
+            name: linux+x86-64
+          - os: [self-hosted, macOS, ARM64]
```

Comment:
regression (we set this to `ubuntu-latest` to prevent hogging the self-hosted runners for work that is not platform specific).

Reply:
k I see that we code sign in the bottle step now. Ignore me.